test_runner/src/main/kotlin/ftl/config/ios/IosFlankConfig.kt

```diff
@@ -25,16 +25,42 @@ data class IosFlankConfig @JsonIgnore constructor(
     @set:JsonProperty("test-targets")
     var testTargets: List<String?>? by data
 
+    @set:CommandLine.Option(
+        names = ["--only-test-configuration"],
+        description = [
+            "Constrains a test action to only test a specified test configuration within a test plan " +
+                    "and exclude all other test configurations. (default: run all test configurations)." +
+                    "Flank can  combine  multiple  constraint options, but -only-test-configuration has precedence over -skip-test-configuration. " +
+                    "Each test configuration name must match the name of a configuration specified in a test plan and is case-sensitive."
+        ]
+    )
+    @set:JsonProperty("only-test-configuration")
+    var onlyTestConfiguration: String? by data
+
+    @set:CommandLine.Option(
+        names = ["--skip-test-configuration"],
+        description = [
+            "Constrains a test action to skip a specified test configuration " +
+                    "and include all other test configurations. (default: run all test configurations). " +
+                    "Flank can combine  multiple constraint options, but -only-test-configuration has precedence over -skip-test-configuration. " +
+                    "Each test configuration name must match the name of a configuration specified in a test plan and is case-sensitive."
+        ]
+    )
+    @set:JsonProperty("skip-test-configuration")
+    var skipTestConfiguration: String? by data
```

Comment:
Isn't be better to allow a list of configurations instead of just one?
In other words:
As a user, If I have a collection of configuration, I would like to skip part of them, not only one.

Reply:
You're right IMO it's better to allow list of configurations. We can change that. 
But, I tried to match Xcode's interface - if you try to run iOS tests from command line, xcodebuild tool is exposing those two parameters: skip-test-configuration, only-test-configuration - that's why I introduced them here

Reply:
OK, If iOS developers are familiar with those options we should keep convention.